change/@fluentui-react-charting-7ed503a2-d9d1-4679-8074-c67eb83161e1.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28504&commitId=eb8fbaa23cac4a93c8d67f32f4ec14a13e93df5a&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-PieChart</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-PieChart.Basic.default.chromium.png|3803| Changed|
|react-charting-PieChart.Basic - Dark Mode.default.chromium.png|4061| Changed|
|react-charting-PieChart.Basic - RTL.default.chromium.png|3805| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Reply:
Minor noise